boards/nxp/common/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NXP_SPSDK_IMAGE
+	bool "Generate bootable firmware image with SPSDK"
```

Comment:
spsdk comes with version, so it is better to add a version check

Reply:
Let me limit it to verified versions in scripts/requirements-extras.txt